src/pages/GiveParty/PartyProfileContent.tsx

```diff
@@ -0,0 +1,180 @@
+import { fetchCoSoul } from 'features/colinks/fetchCoSouls';
+import { artWidthMobile } from 'features/cosoul';
+import { anonClient } from 'lib/anongql/anonClient';
+import { useQuery } from 'react-query';
+import { NavLink } from 'react-router-dom';
+import { CSS } from 'stitches.config';
+
+import { webAppURL } from 'config/webAppURL';
+import { Farcaster, Links, Wand } from 'icons/__generated';
+import { CoSoulItemParty } from 'pages/CoSoulExplorePage/CoSoulItemParty';
+import { coLinksPaths } from 'routes/paths';
+import { Button, Flex, Link, Text } from 'ui';
+
+import { PartyProfileGives } from './PartyProfileGives';
+import { PartyProfileHeader } from './PartyProfileHeader';
+
+const QUERY_KEY_PARTY_PROFILE = 'partyProfile';
+
+export const PartyProfileContent = ({
+  address,
+  css,
+}: {
+  address: string;
+  css?: CSS;
+}) => {
+  const { data, isLoading: fetchCoLinksProfileIsLoading } = useQuery(
+    [QUERY_KEY_PARTY_PROFILE, address, 'profile'],
+    () => fetchCoLinksProfile(address!)
+  );
+  const { data: cosoul } = useQuery(
+    [QUERY_KEY_PARTY_PROFILE, address, 'cosoul'],
+    async () => {
+      return fetchCoSoul(address!);
+    }
+  );
+
+  const targetProfile = data as PublicProfile;
+  const appURL = webAppURL('colinks');
+  const castProfileUrl = `https://warpcast.com/~/compose?text=${appURL}/giveparty/${address}&embeds[]=${appURL}/giveparty/${address}`;
+
+  const { data: details } = useQuery(
+    ['fc_profile', targetProfile?.id],
+    async () => {
+      const { farcaster_accounts_by_pk: farcaster } = await anonClient.query(
+        {
+          farcaster_accounts_by_pk: [
+            {
+              profile_id: targetProfile.id,
+            },
+            {
+              username: true,
+            },
+          ],
+        },
+        {
+          operationName: 'fc_profile',
```

Comment:
more specific operation name so the logs indicate this is the file originating this request